cypress/jenkins/README.md

```diff
@@ -0,0 +1,122 @@
+
+Basically what it does is a setup of a Linux node at a user-level of the software needed to execute the execution script located in the Dashboard tests [Corral Package](https://github.com/rancherlabs/corral-packages).
+
+It configures a `WORKSPACE` folder in the *PATH* to add the necessary binaries.
+
+In Jenkins executors `WORKSPACE` is a predefined variable to a temporary folder in the *jenkins* user home.
+
+- Golang - To build the corral package images.
+- Corral - For executng the corral package to build the remote Node for the tests
+- yq - For updating the corral package configuration values
+
+If the `JOB_TYPE` is `recurring` that will make use of the `rancher` package to spin up a Rancher server for the dashboards tests to run on it.
+
+## Corral configuration
+
+The initialization script makes use of the `corral config vars` option to set the dashboard tests corral package values.
+
+The goal is to have logic in the script for configuring both a Rancher instance and the Dashboard tests node that runs:
+
+ `Dashboard tests node -> Rancher setup`
+
+## Run locally - needs remote aws provider
+
+It is possible to run this locally or in a remove Linux instance.
+
+There's however required environment variables set for configuring the script, download the binary dependencies and execute corral.
+
+This design is mainly due to how the Jenkins Jobs manage the configuration using environment variables.
+
+Some environment have default values thus making them optional others like the following are required:
+
+- `WORKSPACE`
+  - This is defined in Jenkins but it isn't a system variable on Linux.
+- `AWS_ACCESS_KEY_ID`
+- `AWS_SECRET_ACCESS_KEY`
+- `AWS_REGION`
+- `AWS_ZONE_ID`
+- `AWS_DOMAIN`
+- `AWS_AMI`
+- `AWS_SECURITY_GROUP`
+- `AWS_SUBNET`
+- `AWS_VPC`
+- `AWS_VOLUME_TYPE`
+- `AWS_VOLUME_IOPS`
+- `AWS_SSH_USER`
+- `AWS_INSTANCE_TYPE`
+- `RANCHER_TYPE`
+  - This var defines the logic of how to execute the tests by the corral package.
+- `CYPRESS_TAGS`
+
+*For more variables or variables updates take a look at the [init.sh](./init.sh) script.*
+
+Folder structure:
+
+- `WORKSPACE`
+  - `corral-packages` - cloned repo folder.
+  - `dashboard` - cloned repo folder
+  - `bin` - folder with binaries and set in the `PATH`
+
+From `WORKSPACE`:
+
+`dashboard/cypress/jenkins/init.sh`
+
+The expected and default Dashboard corral package image path is:
+
+`${WORKSPACE}/corral-packages/dist/aws-dashboard-tests-t3a.xlarge`
+
+That is generated by `make` during the `init.sh` execution.
+
+### Use run.sh from the corral package directly
+
+The main `run.sh` script has all the logic to directly run the tests without creating the remote AWS node. However that's not fully tested and some modifications might be required to make it work.
+
+## CORRAL PACKAGE
+
+The Dashboard Tests corral package basically create an `AWS` ephimeral node to execute the UI tests on a `cypress` docker container.
```

Comment:
if you make a change to this PR it's worth updating `ephimeral` --> `ephemeral` x3. otherwise it can be left